plugins/available/wttr.plugin.bash

```diff
@@ -0,0 +1,39 @@
+function wttr() {
+if [ -n "$1" ]; then
+    if [ "$(ping -c 1 wttr.in)" ]; then
```

Comment:
Please use the Editorconfig plugin for your IDE/editor - code should be indented with 2 spaces - please see the `.editorconfig` file in the root of the project.